<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSouhait</class>
 <widget class="QWidget" name="WidgetSouhait">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>89</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>89</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>89</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2">
      <widget class="QLabel" name="label_EtiquetteSurfaceJardinSouhaitee">
       <property name="font">
        <font>
         <family>Liberation Sans</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Surface jardin souhaitée</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_BudgetMax">
       <property name="font">
        <font>
         <family>Liberation Sans</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>BudgetMax</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_SurfaceHabitableSouhaitee">
       <property name="font">
        <font>
         <family>Liberation Sans</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>SurfaceH</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_EtiquetteBudgetMax">
       <property name="font">
        <font>
         <family>Liberation Sans</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Budget maximum</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_SurfaceJardinSouhaitee">
       <property name="font">
        <font>
         <family>Liberation Sans</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>SurfaceJ</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_EtiquetteVille">
       <property name="font">
        <font>
         <family>Liberation Sans</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ville(s)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_EtiquetteSurfaceHabitableSouhaitee">
       <property name="font">
        <font>
         <family>Liberation Sans</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Surface habitable souhaitée</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_AccederVilles">
       <property name="text">
        <string>Ville(s)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>60</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_modifierSouhait">
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>61</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>71</width>
         <height>61</height>
        </size>
       </property>
       <property name="icon">
        <iconset resource="ressources.qrc">
         <normaloff>:/app/souhait48</normaloff>:/app/souhait48</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>46</width>
         <height>46</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_supprimerSouhait">
       <property name="maximumSize">
        <size>
         <width>71</width>
         <height>61</height>
        </size>
       </property>
       <property name="icon">
        <iconset resource="ressources.qrc">
         <normaloff>:/app/annuler48</normaloff>:/app/annuler48</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>46</width>
         <height>46</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
